probe/endpoint/reporter.go

```diff
@@ -213,6 +176,10 @@ func (r *Reporter) Report() (report.Report, error) {
 				namespaceID = strconv.FormatUint(conn.Proc.NetNamespaceID, 10)
 			}
 
+			if r.ebpfEnabled && !r.ebpfTracker.initialized {
+				r.ebpfTracker.handleFlow("connect", tuple, int(conn.Proc.PID), namespaceID)
```

Comment:
Exactly. I'm adding the ipv6 connections too btw, even though the ebpf doesn't handle them (to-do).